frontend/src/views/mapper/CourseTray.js

```diff
@@ -184,11 +184,11 @@ const CourseTray = ({
     }
   }, [courseTrayOpen, courses, courseLinks])
 
-  const createCourseLink = useMutation(CREATE_COURSE_LINK, {
+  const [createCourseLink] = useMutation(CREATE_COURSE_LINK, {
     update: cache.createCourseLinkUpdate(workspaceId, activeCourseId)
   })
 
-  const deleteCourseLink = useMutation(DELETE_COURSE_LINK, {
+  const [deleteCourseLink] = useMutation(DELETE_COURSE_LINK, {
     update: cache.deleteCourseLinkUpdate(workspaceId, activeCourseId)
```

Comment:
This updates the cache, but the query data doesn't change in the MapperView -> doesn't render the new state